src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.Arrays;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private Bulldozer bulldozer = new Bulldozer();
+    private Bulldozer bulldozer1 = new Bulldozer();
+    private Bulldozer bulldozer2 = new Bulldozer();
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. It's recommended to make these fields private and provide getters and setters for accessing them.